tests/meta/orders/test_meta_orders_status.py

```diff
@@ -103,18 +103,22 @@ def test_sarscov2_samples_to_status(sarscov2_order_to_submit):
 
     # THEN first sample should contain all the relevant data from the microbial order
     sample_data = data["samples"][0]
-    assert sample_data.get("priority") in "research"
-    assert sample_data["name"] == "all-fields"
     assert sample_data.get("internal_id") is None
-    assert sample_data["organism_id"] == "SARS CoV-2"
-    assert sample_data["reference_genome"] == "NC_111"
+    assert sample_data.get("priority") in "research"
```

Comment:
why is this one handled differently to other values?

Reply:
There was probably a reason at some point, I'll change it to get consistency